### PR TITLE
Added additional instructions to INSTALLATIONS.md

### PR DESCRIPTION
**Pull request by Keartroth**
### Added additional instructions to INSTALLATIONS.md
- Moved VS Code Settings up so that it is underneath VS Code installation
- Added a script to allow students to always open a terminal window in the last directory visited
- Added a section that gives instructions for students to check that everything has been installed

### CODE REVIEW

File: book-1-orientation/chapters/INSTALLATIONS.md [hunk 1]
```diff
@@ -139,6 +156,36 @@ If you haven't already set up Visual Studio Code to format your code and autosav
...
+
+## Installation Check
+Now that you have finished the installation process, please double check that you have not missed something. Check that you have the following programs by pressing your Windows key (Windows keyboard) or Command key (Apple keyboard) to open the Start menue, and begin typing in the name of the following progams to see that they are installed:
+
+```ssh
+Zoom
+Slack
+Google Chrome
+Mozilla Firefox (if you installed it)
+Visual Studio
```
Comment (askingalot) [inline book-1-orientation/chapters/INSTALLATIONS.md:168]: You should separate `Visual Studio Code` from `Visual Studio`. List them twice. I want to call out that they are two different programs ever chance we get.

File: book-1-orientation/chapters/INSTALLATIONS.md [hunk 2]
```diff
@@ -139,6 +156,36 @@ If you haven't already set up Visual Studio Code to format your code and autosav
...
+- check for both Visual Studio & Visual Studio Code
+Postman
+Windows Terminal
```
Comment (askingalot) [inline book-1-orientation/chapters/INSTALLATIONS.md:171]: I take back what I said about SQL Server. They should be able to see `SQL Server 2019 Configuration Manager` as an installed program.

File: book-1-orientation/chapters/INSTALLATIONS.md [hunk 3]
```diff
@@ -139,6 +156,36 @@ If you haven't already set up Visual Studio Code to format your code and autosav
...
+git --version
+node -v
+serve -v
+json-server -v
+dotnet --version
+ls -al ~/.ssh
+git config --list
+- check user.email, user.name, pull.rebase, & init.defaultBranch
+type dnignore
+type cd
```
Comment (askingalot) [inline book-1-orientation/chapters/INSTALLATIONS.md:186]: I don't think these instructions make sense

File: book-1-orientation/chapters/INSTALLATIONS.md [hunk 4]
```diff
@@ -139,6 +156,36 @@ If you haven't already set up Visual Studio Code to format your code and autosav
...
+Mozilla Firefox (if you installed it)
+Visual Studio
+- check for both Visual Studio & Visual Studio Code
+Postman
+Windows Terminal
+```
+
+Finally, run the following commands in Windows Terminal to check that the following are installed or set correctly:
+
+```ssh
```
Comment (askingalot) [inline book-1-orientation/chapters/INSTALLATIONS.md:176]: Would you mind separating these into individual code blocks and providing example output?

**Review by askingalot — CHANGES_REQUESTED** [review]

File: book-1-orientation/chapters/INSTALLATIONS.md (hunk 3 above)
Comment (Keartroth) [reply book-1-orientation/chapters/INSTALLATIONS.md:186]: I've changed the wording a bit. Let me know if it is still obscure.

File: book-1-orientation/chapters/INSTALLATIONS.md [hunk 5]
```diff
@@ -139,6 +156,98 @@ If you haven't already set up Visual Studio Code to format your code and autosav
...
+Visual Studio Code
+Windows Terminal
+Mozilla Firefox (if you installed it)
+SQL Server 2019 Configuration Manager
+```
+
+### Installation Check: Terminal Commands
+Finally, run the following commands without quotes in Windows Terminal to check that the following are installed or set correctly:
+
+```ssh
```
Comment (askingalot) [inline book-1-orientation/chapters/INSTALLATIONS.md:181]: These items should look something like this

```sh
$ git --version
git version 2.28.0.windows.1
```

**Review by askingalot — CHANGES_REQUESTED** [review]

File: book-1-orientation/chapters/INSTALLATIONS.md (hunk 5 above)
Comment (Keartroth) [reply book-1-orientation/chapters/INSTALLATIONS.md:181]: Andy, I notice that a number of the earlier terminal commands are missing the $. Should I go back and add them in? An example would be line 77.

```sh
$ git config --global init.defaultBranch main
```

**Pull request merged**